### PR TITLE
159 161 remove none filters+multiselect native language

### PR DESCRIPTION
**Pull request by KaterynaHerasymiva**

### CODE REVIEW

File: frontend/src/app/shared/components/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -57,14 +57,14 @@ export class DropdownComponent extends BaseComponent implements OnInit, OnDestro
...
+        if (this.selectedItems[0] && this.selectedItems.length === 1) {
             const fullItem = this.data.find(l => l.title === this.selectedItems[0]);
 
             return fullItem && fullItem.subtitle ?
                 `${fullItem.title}: ${fullItem.subtitle}` :
                 fullItem?.title;
         }
 
-        return 'None';
+        return this.btnLabel;//
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/dropdown/dropdown.component.ts:68]: Remove comment

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by SidletskyiOlexandr — DISMISSED** [review]

File: frontend/src/app/shared/components/dropdown/dropdown.component.ts (hunk 1 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/dropdown/dropdown.component.ts]: fixed

**Review by SidletskyiOlexandr — DISMISSED** [review]

**Review by Anatolii-Ivanov — DISMISSED** [review]

File: frontend/src/app/shared/components/dropdown/dropdown.component.html [hunk 2]
```diff
@@ -5,10 +5,13 @@
                 {{selectedItems.length || 0}}
             </div>
         </div>
-        <div class="dropdown-btn-title">
+        <div *ngIf="selectedItems.length === 0" class="dropdown-btn-title" >
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/dropdown/dropdown.component.html:8]:
```suggestion
        <div *ngIf="!selectedItems.length" class="dropdown-btn-title" >
```

File: frontend/src/app/shared/components/dropdown/dropdown.component.html [hunk 3]
```diff
@@ -5,10 +5,13 @@
...
             {{getFullTitle()}}
         </div>
-        <div *ngIf="btnLabel" class="dropdown-btn-subtitle">
+        <div *ngIf="selectedItems.length > 0" class="dropdown-btn-title-when-selected" >
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/dropdown/dropdown.component.html:11]:
```suggestion
        <div *ngIf="selectedItems.length" class="dropdown-btn-title-when-selected" >
```

File: frontend/src/app/shared/components/dropdown/dropdown.component.html [hunk 4]
```diff
@@ -5,10 +5,13 @@
...
+            {{getFullTitle()}}
+        </div>
+        <div *ngIf="selectedItems.length > 0" class="dropdown-btn-subtitle">
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/dropdown/dropdown.component.html:14]:
```suggestion
        <div *ngIf="selectedItems.length" class="dropdown-btn-subtitle">
```

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**